### PR TITLE
RandomGenerateTransaction

### PR DESCRIPTION
**Pull request by cmdhema**
**Description**:

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: HIP/hip-350.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+hip: TBD
```
Comment (mgarbs) [inline HIP/hip-350.md:2]:
```suggestion
hip: 351
```

File: HIP/hip-350.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+title: Add RandomGenerateTransaction
+author: Jaewook Kim <jaewook90.kim@lge.com>, Leemon Baird <leemon@swirlds.com>
+type: Standards Track
+category: Service
+needs-council-approval: Yes
+status: Draft
```
Comment (mgarbs) [inline HIP/hip-350.md:8]:
```suggestion
status: Review
```
Please update the status when its ready to be merged.

**Review by cmdhema — COMMENTED** [review]
Approve

**Review by sergmetelin — APPROVED** [review]
LGTM

**Pull request merged**